src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -400,6 +467,7 @@ public InFilter(InFilter.Operator operator, String path, List<Object> arrayValue
       super(path);
```

Comment:
nit: Constructor should probably come after other methods in the class (I know it was already in sort of odd place and not changed here but was bit confusing)